cfg_mgmt/reporting.py

```diff
@@ -201,6 +201,10 @@ def storage_summary(element_storage: str) -> cmm.CfgStorageSummary:
     yield from storage_summaries.values()
 
 
+def cfg_element_status_name(status: CfgElementStatusReport):
```

Comment:
this one I would have considered a good candidate for a method/@property; however, I am also fine w/ having separate functions, generally speaking.